features/works/work_edit.feature

```diff
@@ -144,6 +144,22 @@ Feature: Edit Works
     Then I should see "ex_friend" within ".byline"
       And I should not see "coolperson" within ".byline"
 
+  Scenario: User applies a coauthor's work skin to their work
+    Given the following activated users with private work skins
+        | login       |
+        | lead_author |
+        | coauthor    |
+        | random_user |
+      And I coauthored the work "Shared" as "lead_author" with "coauthor"
+      And I am logged in as "lead_author"
+    When I edit the work "Shared"
+    Then I should see "Lead Author's Work Skin" within "#work_work_skin_id"
```

Comment:
For Cucumber tests, we generally prefer to use the label text rather than the field id or name to better reflect what the user sees.

Reply:
This ask would be pretty straightforward if I was looking at the actual value of the field, but testing what select options are available (not selected) with only a given label is surprisingly complicated.

I updated line 159 to use the label, since I'm just selecting in that line. But for lines 156-158, how important is it to create a new step that lets people test the presence of select options by label?

Reply:
Oh, sorry about that! I thought `The field labeled "Label" should contain "option"` would work do that, but it looks like it can't handle select fields with more than one option. I think we can live with the field id here.